Automattic/wp-calypso
# Photos picker: Replace config feature flag with the API fetched feature status

Related to D165605-code

## Proposed Changes

* Update store to support google-photos-picker feature status
* Remove non-used env config feature flag
* Replace the logic for checking the fetched feature status

## Why are these changes being made?


* Recently, we introduced an endpoint for fetching the feature status to have a single source of truth since we need it on both the FE and BE sides.

## Testing Instructions

* On your sandbox, update feature flag `const IS_PICKER_ENABLED = true;` in `google-photos.php` file
* Go to `/media/SITE_SLUG`
* Check if everything is the same as before this change
---
* Without the sandbox change, everything should be the same

## Pre-merge Checklist
